src/dymaptic.GeoBlazor.Core/Components/Layers/Layer.cs

```diff
@@ -170,7 +170,7 @@ public async Task Load(CancellationToken cancellationToken = default)
         IJSObjectReference arcGisJsInterop = await GetArcGisJsInterop();
         JsLayerReference = await arcGisJsInterop.InvokeAsync<IJSObjectReference>("createLayer", 
             // ReSharper disable once RedundantCast
-            cancellationToken, (object)this, true);
+            cancellationToken, (object)this, true, View?.Id);
```

Comment:
in the samples, this would be the mapView id?  Would it be useful to update more of the samples to have mapView ids?...in future releases or could this be more of a suggestion in the documentation.  Something along the lines: as a developer creates more complex mapViews, Layers, etc; implementing some of these classes such as popups will require more elements to have designated Ids for the best customization.

Reply:
I'm only passing it here because it's needed to register the popup action from popup templates to the view.